src/analysis/retail/priest/shadow/modules/guide/CooldownGraphSubsection.tsx

```diff
@@ -117,13 +156,26 @@ const CoreCooldownsGraph = () => {
     </p>
   );
 
+  if (info!.combatant.hasTalent(TALENTS.VOID_BLAST_TALENT)) {
```

Comment:
quick note since i'm seeing a lot of the `info!` assertion. you can put something like this at the top (after all of the hooks) and then typescript will know that `info` isn't null/undefined:

```ts
if (!info) {
  return null;
}
```

`info` is only ever null/undefined when stuff is still loading. in practice i don't think it will *ever* be null/undefined for a component like this.

(you don't need to fix that here---but future note)